specification/authorization/resource-manager/Microsoft.Authorization/preview/2020-04-01-preview/authorization-RoleAssignmentsCalls.json

```diff
@@ -463,6 +502,12 @@
             "schema": {
               "$ref": "#/definitions/RoleAssignmentListResult"
             }
+          },
+          "400": {
+            "description": "Bad Request - Indicates the request has missing information and can not be completed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
           }
```

Comment:
Also the error response here is causing autorest generation failure - our convention is that the `responses` only list the successful status code that would return something from the service, not these error responses.